ui/hooks/accounts/useMultichainWalletSnapClient.ts

```diff
@@ -52,11 +52,18 @@ export class MultichainWalletSnapClient {
     this.#client = new KeyringClient(new MultichainWalletSnapSender(snapId));
   }
 
-  async createAccount(scope: CaipChainId) {
+  async createAccount(scope: CaipChainId, entropySource?: string) {
     // This will trigger the Snap account creation flow (+ account renaming)
-    const account = await this.#client.createAccount({
-      scope,
-    });
+    const account = await this.#client.createAccount(
+      entropySource
+        ? ({
+            scope,
+            entropySource,
+          } as Record<string, string>)
+        : {
+            scope,
+          },
+    );
```

Comment:
Cosmetic: But I feel like we can avoid the type-cast with: (since we check for `entropySource ?` it will guaranteed to be defined)

```suggestion
    const account = await this.#client.createAccount({
      scope,
      ...(entropySource ? { entropySource } : {})
    });
```